management_app/views/auth.py

```diff
@@ -1,61 +1,84 @@
+import os
+import pathlib
 import functools
 
-from flask import (
-    Blueprint, flash, g, redirect, render_template, request, session, url_for
-)
-from werkzeug.security import check_password_hash, generate_password_hash
-
-from management_app.db import get_db
+import requests
+from flask import Flask, session, redirect, request, render_template, Blueprint, url_for
+from google.oauth2 import id_token
+from google_auth_oauthlib.flow import Flow
+from pip._vendor import cachecontrol
+import google.auth.transport.requests
 
 bp = Blueprint('auth', __name__, url_prefix='/auth')
 
-@bp.route('/login', methods=('GET', 'POST'))
-def login():
-    if request.method == 'POST':
-        username = request.form['username']
-        password = request.form['password']
-        db = get_db()
-        error = None
-        user = db.execute(
-            'SELECT * FROM user WHERE username = ?', (username,)
-        ).fetchone()
-
-        if user is None:
-            error = 'Incorrect username.'
-        elif not check_password_hash(user['password'], password):
-            error = 'Incorrect password.'
-
-        if error is None:
-            session.clear()
-            session['user_id'] = user['id']
-            return redirect(url_for('index'))
-
-        flash(error)
-
-    return render_template('auth/login.html')
-
-@bp.route('/logout')
-def logout():
-    session.clear()
-    return redirect(url_for('index'))
+os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "1"
+
+GOOGLE_CLIENT_ID = "559245110133-8bog3pa5k75ih9q1fkkauiv8latmbgjf.apps.googleusercontent.com"
```

Comment:
The value of GOOGLE_CLIENT_ID can also be put in the `.env` file if possible.

Reply:
I agree. will put it in the `.env` and update the setup instruction in readme.